app/tests/test_crds.py

```diff
@@ -112,6 +115,88 @@ def test_deserialization(self):
         assert crd.metadata.name == "foo"
         assert crd.metadata.uid == "c560d138-a93a-4463-8b44-d7717851a265"
 
+    def test_is_browser_shortcut_enabled_disallowed_on_wildcard_resource(self):
```

Comment:
What is the expected behavior if ports start > end ?

Reply:
@liorr API call would fail but you're right we can fail early with CRD validation